hack/build/ci/generate-release-notes.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+tag=${GITHUB_REF_NAME:-<tag>}
+tag_without_prerelease=${tag%%-*}
+output_file=${OUTPUT_FILE:-CHANGELOG.md}
+pre_release=${PRE_RELEASE:-false}
+
+pre_release_warning="> ⚠️ This is a pre-release, which has no official support by Dynatrace. If you run into issues with this specific release, please open a Github Issue!
+> 
+> Release notes for ${tag_without_prerelease} will be published in our official documentation.
+"
+
+default_notes="### Installation
```

Comment:
The text portions could be put into separate files and used from there.

Reply:
In my opinion it's better to have these templates in one single file.